packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -3883,6 +3915,28 @@ describe('StripeHelper', () => {
       // reset for tests:
       existingCustomer = await createAccountCustomer(existingUid, customer1.id);
     });
+
+    it('expands the tax information if present', async () => {
+      const customer = deepCopy(customer1);
+      const customerSecond = deepCopy(customer1);
+      customerSecond.tax = {
+        location: { country: 'US', state: 'CA', source: 'billing_address' },
+        ip_address: null,
+        automatic_tax: 'supported',
+      };
+      sandbox.stub(stripeHelper, 'expandResource').returns(customer);
+      sandbox
+        .stub(stripeHelper.stripe.customers, 'retrieve')
+        .resolves(customerSecond);
+      const result = await stripeHelper.fetchCustomer(existingCustomer.uid, [
+        'tax',
+      ]);
+      const customerResult = {
+        ...customer,
+        tax: customerSecond.tax,
+      };
+      assert.deepEqual(result, customerResult);
+    });
```

Comment:
I don't see a test for StripeHelper's `createSubscriptionWithPMI` here -- can we assert that it's always passing in the `tax` field now in its existing test?

Reply:
Nevermind this was added in a previous patch.